Source/CISQLite3/Private/SQLiteDatabase.cpp

```diff
@@ -17,7 +17,7 @@ USQLiteDatabase::USQLiteDatabase(const FObjectInitializer& ObjectInitializer)
 
 bool USQLiteDatabase::CreateDatabase(const FString& Filename, bool RelativeToProjectContentDirectory)
 {
-	const FString actualFilename = RelativeToProjectContentDirectory ? FPaths::ProjectContentDir() + Filename : Filename;
+	const FString actualFilename = RelativeToProjectContentDirectory ? FPaths::ConvertRelativePathToFull(FPaths::ProjectContentDir()) + Filename : Filename;

```

Comment:
I remember fighting path issues on Android, but I don't think this should be a problem.
Anyone test this on Android coincidentally? 🤔

Reply:
Haven't tested on anything else but Windows, sorry. I'm not sure why I did this either, but IIRC a relative path didn't work correctly.

I'm sorry for the incredibly low quality of the MR - I just managed to make it work on my machine and decided to share ;p

Thanks for your time!

Reply:
No worries, sharing is always valuable, maybe someone using this on Android will pick it up and verify 👍